packages/wallet-adapter-tailwind-ui/src/WalletDropdown.tsx

```diff
@@ -0,0 +1,115 @@
+import { WalletReadyState } from '@aptos-labs/wallet-adapter-core';
```

Comment:
let's not have this package use `core` as a dependency and instead import and export `WalletReadyState` from the `react` package, same as we do for `WalletName` type.
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-react/src/useWallet.tsx#L13

I can handle it and publish a new version for `react` package that exports this type.

Reply:
@WEB3Zer0 https://github.com/aptos-labs/aptos-wallet-adapter/pull/36
once approved I can bump the version and you can update and use it